src/main.ts

```diff
@@ -481,11 +493,10 @@ export default class NewZettel extends Plugin {
   }
 
   getZettels(): TFile[] {
+    const fileToId = (file: TFile) => this.fileToId(file.basename)
     return this.app.vault.getMarkdownFiles().filter(file => {
-      const ignore = /^[_layouts|templates|scripts]/;
-      return
-        this.fileToId(file.basename) !== '' &&
-        file.path.match(ignore) == null;
+      const ignore = !file.path.match(/^[_layouts|templates|scripts]/);
+      return ignore && fileToId(file) !== '';
```

Comment:
I think there is a missing `this.` here.

Reply:
I agree